### PR TITLE
IBX-4442: Remove old Commerce from user documentation

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | [IBX-4442](https://issues.ibexa.co/browse/IBX-4423) |
| Versions      | master |

Remove old Commerce from user documentation to clear ground for the new design.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: plugins.yml [hunk 1]
```diff
@@ -31,14 +31,12 @@ plugins:
             'shop_administration/manage_products.md': 'pim/products.md'
             'shop_administration/product_categories.md': 'pim/work_with_product_categories.md'
             'shop_administration/catalogs.md': 'pim/work_with_catalogs.md'
-            'shop_administration/cockpit.md': 'shop_administration/discover_dashboard.md'
+            'shop_administration/cockpit.md': 'cdp/customer_portal.md'
```
Comment (adamwojs) [inline plugins.yml:34]: CDP usually refers to "Customer Data Platform" and have nothing in common with customer portals/corporate accounts. Are you sure it's the right place for "Company self-registration", "Customer Portal account" articles ?

Comment (dabrt) [reply plugins.yml:34]: I see your pint now. Yes, I will move this content to its own category of customer management.

**Comment by dabrt** [comment]
@adamwojs Please sign off the PR

**Review by MagdalenaZuba — APPROVED** [review]

File: plugins.yml [hunk 2]
```diff
@@ -31,14 +31,12 @@ plugins:
...
+            'shop_administration/cockpit.md': 'customer_management/customer_portal.md'
             'search.md': 'search/search_for_content.md'
             'personalization/enabling_personalization.md': 'personalization/enable_personalization.md'
             'personalization/perso_configuration.md': 'personalization/configure_personalization.md'
             'personalization/previewing_scenario.md': 'personalization/preview_scenario_results.md'
             'personalization/integrating_results.md': 'personalization/integrate_scenario_results.md'
             'personalization/dashboard.md': 'personalization/review_perso_performance.md'
-            'shop_administration/manage_users.md': 'shop_administration/manage_customers.md'
+            'shop_administration/manage_users.md': 'customer_management/manage_customers.md'
             'site_organization/site_factory.md': 'website_organization/work_with_sites.md'
```
Comment (MagdalenaZuba) [inline plugins.yml:42]: Add redirects to removed pages.

**Pull request merged**